setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name="pylint-pytest",
-    version="1.1.3",
+    version="1.1.4",
```

Comment:
_These changes_ probably belong to the 2.0 scope.

_See https://github.com/pylint-dev/pylint-pytest/milestone/2 and https://github.com/pylint-dev/pylint-pytest/pull/8#issuecomment-1763332031_